documentation/modules/ROOT/pages/developer-resources.adoc

```diff
@@ -1,11 +1,15 @@
-= Solution Patterns: Template Name
+= Building Apps around the Event Mesh
 :sectnums:
 :sectlinks:
 :doctype: book
 
 
-= Developer Resources
-
-* Add link to the git repo of the code
-* Add links to whitepapers, ebooks, learning paths, product pages
+== Developer Resources
 
+* https://github.com/cardil/cabs-usvc[cardil / cabs-usvc] _(The example code used in this solution)_
```

Comment:
mention that this is the source repository in the link name.

Reply:
Hmm. What do you mean? The description is already in the parenthesis. Do you think of something like the following?

```asciidoc
https://github.com/cardil/cabs-usvc[The example code used in this solution]
```